sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityAnalysisInterface.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.sensitivity_computation;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Crac;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.sensitivity.SensitivityComputationParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * A computation engine dedicated to the systematic sensitivity analyses performed
+ * in the scope of the LinearRao.
+ *
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
+ */
+public class SystematicSensitivityAnalysisInterface {
```

Comment:
Not a big fan of the naming I think it is still a bit too long but why not :)

Reply:
changed to SystematicSensitivityInterface (and SystematicSensitivityResult and SystematicSensitivityService)